bundles/org.openhab.binding.opensprinkler/README.md

```diff
@@ -49,9 +49,10 @@ Bridge opensprinkler:http:http [stations=8, refresh=60] {
 
 The following channel is supported by the station thing.
 
-| Channel Type ID | Item Type |    | Description                                             |
-|-----------------|-----------|----|---------------------------------------------------------|
-| stationState    | Switch    | RW | This channel indicates whether station 01 is on or off. |
+| Channel Type ID    | Item Type |    | Description                                             |
+|--------------------|-----------|----|---------------------------------------------------------|
+| stationState       | Switch    | RW | This channel indicates whether station 01 is on or off. |
+| remainingWaterTime | Number    | R  | The time the station remains to be open, in seconds.    |
```

Comment:
Why not use `Number:Time`? The unit can be set by the user then and will automatically be converted.

Reply:
Makes sense :)